ci/docker/release-rocm533.yaml

```diff
@@ -10,7 +10,7 @@
 
 spack:
   specs:
-  - dla-future@develop +rocm +miniapps +ci-test
+  - dla-future@develop +rocm amdgpu_target=gfx90a:xnack- +miniapps +ci-test
```

Comment:
Actually, I had to.

I'm not sure why, but when I tried to concretise it without it, it complained that `+rocm` and `amdgpu_target=none` are not compatible, which is what we want, but it is strange that spack concretiser does not pick our preference as you said.

If you have any suggestion, I will happily try it (we can easily do some test in my vm).

Reply:
Unfortunately I don't really have any suggestions. I guess it could be related to the require/prefer distinction, but I don't really know. This is not really a blocker for this PR.